### PR TITLE
Update to 0.0.4 release

### PR DESCRIPTION
**Pull request by snowman2**
@conda-forge-admin, please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{% set version = "0.0.2" %}
+{% set version = "0.0.3" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Can we un-comment the entrypoints and CLI tests with this release.

Comment (snowman2) [reply recipe/meta.yaml]: Sure, I will give it a go.

**Comment by snowman2** [comment]
Oh man, I get it now. 🤦‍♂️

**Comment by ocefpaf** [comment]
Thanks!

**Comment by snowman2** [comment]
Not a problem - thanks for setting up `geocube`!

**Pull request merged**